### PR TITLE
fixed landing

### PR DESCRIPTION
**Pull request by MarlAndre**
I changed the position of the form as discussed during the standin
landing iphone12
g

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -22,10 +22,10 @@ class Item < ApplicationRecord
   has_many :requests, dependent: :destroy
   has_many_attached :photos
 
-  # Validations
-  validates_presence_of :user_id, :photos, :description, :expiration_date, :item_type, :status, :name
+  validates_presence_of :user_id :description, :expiration_date, :item_type, :status, :name
   validates :description, length: { minimum: 7 }
```
Comment (stshayna) [inline app/models/item.rb:26]: @MarlAndre please add back `:photos` in as a validation check, it was a recent change pushed to master by Denzel.

Comment (DenzelBraithwaite) [reply app/models/item.rb:26]: agreed

File: app/views/items/index.html.erb [hunk 2]
```diff
@@ -17,7 +17,9 @@
         <%# <span class="closebtnpopup" data-action="click->postal-code#closeBtnPopup">&times;</span> %>
         <label for="postal_code"></label>
         <input class="form-control" type="text" id="postal_code" name="postal_code" value="<%= params[:postal_code] %>" data-postal-code-target="input" placeholder="Enter your zip code...">
-        <button name="button" id="btn" type="submit" class="btn btn-flat">Submit</button>
+        <div class="d-flex justify-content-center">
+          <button name="button" id="btn" type="submit" class="btn btn-flat">Let start !</button>
```
Comment (DenzelBraithwaite) [inline app/views/items/index.html.erb:21]: For now, we can fix it and put 'Start' or something similar. In the future, we should look into another 'catchy' word.

File: app/models/item.rb [hunk 3]
```diff
@@ -24,7 +24,7 @@ class Item < ApplicationRecord
 
   # Validations
   validates_presence_of :user_id, :photos, :description, :expiration_date, :item_type, :status, :name
-  validates :description, length: { minimum: 7 }
+  validates :description, length: { minimum: 10 }
```
Comment (DenzelBraithwaite) [inline app/models/item.rb:27]: has to be 7 otherwise we have to tweak seed

File: app/views/items/index.html.erb [hunk 4]
```diff
@@ -84,11 +86,10 @@
         <div class="tab-pane fade show active" id="nav-see-items" role="tabpanel" aria-labelledby="nav-see-items-tab">
         <br>
           <!-- Cards-->
-          <div class="cards items" id="cards-items">
+          <div class="cards items" id="cards-items" data-postal-code-target="items">
```
Comment (DenzelBraithwaite) [inline app/views/items/index.html.erb:89]: is this needed ? the id

**Review by DenzelBraithwaite — APPROVED** [review]

**Pull request merged**